nexus/src/app/sagas/snapshot_create.rs

```diff
@@ -200,6 +204,9 @@ declare_saga_actions! {
     FINALIZE_SNAPSHOT_RECORD -> "finalized_snapshot" {
         + ssc_finalize_snapshot_record
     }
+    RELEASE_VOLUME_LOCK -> "volume_unlock" {
```

Comment:
Could we fail during some previous unwind step and never get to this unlock
step (and, leave the volume locked)?

Reply:
Yeah, if the snapshot create saga gets stuck, then the volume remains locked.